itn-eval-metrics.Rmd

```diff
@@ -158,7 +158,23 @@ collabs %>% dplyr::count(Category) %>%
                  plot.margin = unit(c(.75,.5,.5,.5), "cm")) + 
   ggplot2::xlab("")
 
-ggplot2::ggsave(file.path("plots", "itn_collaboration_types.png"), width = 7, height = 5)
+ggplot2::ggsave(file.path("plots", "itn_collaboration_types_all.png"), width = 7, height = 5)
+```
+
+```{r}
+collabs %>% dplyr::filter(ITN_ITCR_or_external == "ITCR") %>% 
```

Comment:
Are ITN values also ITCR funded? If so, would we want to select rows with either ITCR or ITN values?

Reply:
Carrie explained the difference that ITN are the internal people :)